Google.Api.Gax.Grpc/ServiceMetadata.cs

```diff
@@ -0,0 +1,77 @@
+﻿/*
+ * Copyright 2022 Google LLC
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using Google.Protobuf.Reflection;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Google.Api.Gax.Grpc
+{
+    /// <summary>
+    /// Provides metadata about a single service within an API.
+    /// Often most of these aspects will be the same across multiple services,
+    /// but they can be specified with different values in the original proto, so 
+    /// </summary>
+    public sealed class ServiceMetadata
```

Comment:
I'm wondering what value does this add over ApiMetadata? (I know I mentioned per method granularity for specifying transport and this is the closest we can get)